### PR TITLE
Docs fixes

### PR DESCRIPTION
**Pull request by abadger**
##### SUMMARY
Updates to fix warnings in docs building and updates for the recent network module changes

##### ISSUE TYPE
 - Docs Pull Request

##### ANSIBLE VERSION

```
devel
```

### CODE REVIEW

**Review by gundalow — APPROVED** [review]

**Review by dagwieers — APPROVED** [review]
That was an interesting read.

File: docs/bin/plugin_formatter.py [hunk 1]
```diff
@@ -409,16 +409,45 @@ def process_categories(mod_info, categories, templates, output_dir, output_name)
 def process_support_levels(mod_info, templates, output_dir):
     supported_by = {'Ansible Core Team': {'slug': 'core_supported',
                                           'modules': [],
-                                          'output': 'core_maintained.rst'},
+                                          'output': 'core_maintained.rst',
+                                          'blurb': "These are :doc:`modules maintained by the"
```
Comment (bcoca) [inline docs/bin/plugin_formatter.py:413]: should we use 'plugins' instead in view of extending the coverage?

File: docs/bin/plugin_formatter.py [hunk 2]
```diff
@@ -409,16 +431,45 @@ def process_categories(mod_info, categories, templates, output_dir, output_name)
...
                     'Ansible Partners': {'slug': 'partner_supported',
                                          'modules': [],
-                                         'output': 'partner_maintained.rst'},
+                                         'output': 'partner_maintained.rst',
+                                         'blurb': """
+Some examples of :doc:`Certified Modules<partner_maintained>` are those submitted by other
+companies. Maintainers of these types of modules must watch for any issues reported or pull requests
+raised against the module.
+
+The Ansible Core Team will review all modules becoming Certified.  Core Committers will review
```
Comment (dharmabumstead) [inline docs/bin/plugin_formatter.py]: please don't capitalize "committers" or "certified".

File: docs/bin/plugin_formatter.py [hunk 3]
```diff
@@ -409,16 +431,45 @@ def process_categories(mod_info, categories, templates, output_dir, output_name)
...
+
+These modules are currently shipped with Ansible, but might be shipped separately in the future.
+"""},
                     'Ansible Community': {'slug': 'community_supported',
                                           'modules': [],
-                                          'output': 'community_maintained.rst'},
+                                          'output': 'community_maintained.rst',
+                                          'blurb': """
+These are :doc:`modules maintained by the Ansible Community<community_maintained>`.  They **are
+not** supported by The Ansible Core Team or by companies/partners associated to the module.
```
Comment (dharmabumstead) [inline docs/bin/plugin_formatter.py]: Please don't capitalize "The" - should read "the Ansible Core Team"

File: docs/docsite/rst/modules_support.rst [hunk 4]
```diff
@@ -18,46 +18,34 @@ The modules are hosted on GitHub in a subdirectory of the `ansible <https://gith
...
-    List of :ref:`Network Supported<network_supported>` modules
-
 Certified
 `````````
 
-Some examples of Certified modules are those submitted by other companies. Maintainers of these types of modules must watch for any issues reported or pull requests raised against the module.
+Some examples of :doc:`Certified Modules<partner_maintained>` are those submitted by other companies. Maintainers of these types of modules must watch for any issues reported or pull requests raised against the module.
 
-Core Committers will review all modules becoming Certified.  Core Committers will review proposed changes to existing Certified modules once the community maintainers of the module have approved the changes. Core committers will also ensure that any issues that arise due to Ansible engine changes will be remediated.
+The Ansible Core Team will review all modules becoming Certified.  Core Committers will review proposed changes to existing Certified modules once the community maintainers of the module have approved the changes. Core committers will also ensure that any issues that arise due to Ansible engine changes will be remediated.
```
Comment (dharmabumstead) [inline docs/docsite/rst/modules_support.rst]: please don't capitalize "certified" or "committers"

File: docs/templates/plugin.rst.j2 [hunk 5]
```diff
@@ -220,15 +222,13 @@ Status
...
+Maintenance Info
+~~~~~~~~~~~~~~~~
 
-Support Level
-~~~~~~~~~~~~~
-
-This module is maintained by @{ support[metadata.supported_by] }@
-
-For more information on what this means please read :doc:`modules_support`
+For more information about this module’s supportability from Red Hat, please
```
Comment (dharmabumstead) [inline docs/templates/plugin.rst.j2]: Consider revising to "For more information about Red Hat's support for this module..."

File: docs/templates/plugin.rst.j2 [hunk 6]
```diff
@@ -220,15 +222,13 @@ Status
...
+refer to this `Knowledgebase Article<https://access.redhat.com/articles/rhel-top-support-policies>`
```
Comment (dharmabumstead) [inline docs/templates/plugin.rst.j2]: Please revise to "knowledge base article".

**Review by dharmabumstead — CHANGES_REQUESTED** [review]
Overall looks great. A few edit suggestions. Thanks @abadger!

**Comment by abadger** [comment]
Merged to devel

**Pull request merged**